src/schema/types/business.graphql

```diff
@@ -0,0 +1,31 @@
+type Business {
+  id: ID!
+  fantasyName: String!
+  realName: String!
+  vertical: String!
+  CUIT: String!
+  AFIPCondition: AFIPConditionEnum!
+  address: Address!
+}
```

Comment:
I agree with this implementation, but just for being curious, what do you guys think about extracting the Legal Information into its own type? Wouldn't this simplify the Business controller resolvers?

```suggestion
type LegalInformation {
  fantasyName: String!
  realName: String!
  vertical: String!
  CUIT: String!
  AFIPCondition: AFIPConditionEnum!
}

type Business {
  id: ID!
  legalInformation: LegalInformation!
  address: Address!
 }
```

Reply:
I think it shows the underlying structure too much, it makes the implementation easier though.

Reply:
Yeah, I agree with Mauro. If business rules dictate that a business can have only one legal information, then I think the solution implemented by Mauro is better for usage by a client.